index.js

```diff
@@ -1,51 +1,127 @@
 //------------------------Question 1 ---------------------------
-//Is time to create things in the DOM!!. We want you to create a new 'div' Element and append it as a child of the element that has the 'id = a-1' (the element is in index.html file)
-// all right our new element needs style. For that, assign to the new element the className 'square'!. Each time the user clicks on the new element it has to respond changing the displayed text 'x' to 'o'
-
-
-
+//Is time to create things in the DOM!!.
+//We want you to create a new 'div' Element and append it as a child of the element that has the 'id = a-1'
+//(the element is in index.html file)
+// all right our new element needs style.
+//For that, assign to the new element the className 'square'!.
+//Each time the user clicks on the new element it has to respond changing the displayed text 'x' to 'o'
 
 //resolve // QUESTION 1 here
+let q1 = document.querySelector("#a-1");
+let newDiv = document.createElement("div");
 
+q1.appendChild(newDiv);
+newDiv.className = "square";
+newDiv.innerHTML = "x";
 
-
+newDiv.addEventListener("click", function () {
+  if ((newDiv.innerHTML = "x")) {
+    newDiv.innerHTML = "o";
+  } else {
+    newDiv.innerHTML = "x";
+  }
+});
 
 //------------------------Question 2 ---------------------------
 //Write a JavaScript program to remove items from a dropdown list.
-//In order to create the list we are providing you with the array bellow, those items are the ones you need to add to the dropdown list as the select options
+//In order to create the list we are providing you with the array below,
+//those items are the ones you need to add to the dropdown list as the select options
 //you also need to create a button that makes the remove action of the element selected
 // append the new list as a child of the element with the 'id = a-2'
 // append the button as a child of the element with the 'id = a-2'
-const colors = [ 'red' , 'white', 'black' , 'green' , 'orange'];
-
-
-
+const colors = ["red", "white", "black", "green", "orange"];
 
 //resolve // QUESTION 2 here
 
+const question2 = () => {
+  let q2 = document.querySelector("#a-2");
+  let innerHTML = '<select id="colors">';
```

Comment:
The variable innerHTML could be confused with the method innerHTML. It would be better to change the name of this variable.